packages/utils/lib/json.ts

```diff
@@ -37,10 +38,11 @@ export function stringifyAndTruncateValue(value: any, maxSize: number = MAX_LOG_
         return 'undefined';
     }
 
-    let msg = typeof value === 'string' ? value : truncateJsonString(stringifyObject(value), maxSize);
+    const msg = typeof value === 'string' ? value : truncateJsonString(stringifyObject(value), maxSize);
 
-    if (msg && msg.length > maxSize) {
-        msg = `${msg.substring(0, maxSize)}... (truncated)`;
+    const truncated = truncateBytes(msg, maxSize);
+    if (truncated !== msg) {
+        return `${truncated}... (truncated)`;
```

Comment:
Note if its truncated at max length then its now above max length again with the added msg

Reply:
that's how it was before. I imagine that's why the `MAX_LOG_PAYLOAD` is `99_000` when the actual limit is 100k

Reply:
Yes indeed 👍🏻